### ISSUE TITLE
SSD single shot detector: DefaultBoxes(), SSDLoss()

### ISSUE THREAD

**Issue by Arbaaz-Shaikh** [Deci-AI/super-gradients#1671]
### 📚 Documentation Improvement

But what about SSD model.
How to calculate the SSDLoss(dboxes= DefaultBoxes)

how to calculate this.

is this a bug?

dataset Prep:

Model arch= ssd_lite_mobilenet_v2
Batch size = 8
Max epochs= 2
Workers= 1
Classes= [ list of string of 140 classes]
Input Image dimensions = 640*640
Note: i tried with 320*320 wont work.

train_data = coco_detection_yolo_format_train (
dataset_params={
"data_dir": dataset_params ["data_dir"],
"images_dir": dataset_params ["train_
_images_dir"],
"labels _dir": dataset_params ["train_labels_dir"],
"classes": dataset_params ["classes"],

dataloader_params={
'batch
_size': BATCH_SIZE,
'num_workers' : WORKERS
val_data = coco_detection_yolo_format_val (
dataset_params=‹
"data_dir": dataset_params l"data_dir"],
"images_dir": dataset_params ["val
_images_dir"],
"labels_dir": dataset_params ["val_labels_dir"],
"classes": dataset_params ["classes"],
dataloader_params={
'batch_size': BATCH_SIZE,
'num_workers': WORKERS
｝，

Training parameters prep:

Set the training parameters

train_params = {
'max_epochs': MAX_EPOCHS,
'Ir_mode': 'CosineLRScheduler',
'optimizer': 'SGD'
optimizer_params'! {'momentum': 0.9, 'weight_decay': 0.0005, 'nesterov': True},
'loss': SSDLoss (dboxes= DefaultBoxes) ,
'valid
_metrics_list': [
{'DetectionMetrics': {'post_prediction_callback': SSDPostPredictCallback(),
'num_cls': len (dataset_params ['classes'])}}1,
'metric_to_watch': mAP@0.50:0.95',
'average_best_models': True,
'initial_lr': 0.01,
'warmup_momentum' : 0.8,
'warmup_bias_lr': 0.1,

Any help will be appreciated:

What are the suitable  DefaultBoxes values for my case ?

Appreciate your efforts.

![IMG_1792](https://github.com/Deci-AI/super-gradients/assets/52887465/9b1e6630-cbea-4f96-9217-bd2661761dfb)

**Comment by Arbaaz-Shaikh** [comment]
ssd_anchors.yaml file can help to resolve this issue
